handlers/Home/components/Services/index.js

```diff
@@ -0,0 +1,54 @@
+/* @flow */
+require('./styles.css');
+
+import React from 'react';
+
+const SERVICES = [
+  ['Product Strategy', 'internet-of-things', '/public/images/blogimg_all.png', 'Through interviews, joint ideation, storyboards and estimation, we give you an informed, fresh perspective to help you prioritize concepts for further investment.'],
+  ['UX Design', 'rapid-innovation', '/public/images/blogimg_all.png', 'Problems cannot be solved with the same thinking that created them. We give you fresh, technological ideas to help push the bounds of what is possible.'],
+  ['Prototype', 'proof-of-concept', '/public/images/blogimg_all.png', 'You’ve got to show them something. Your boss, the executives -- they need more than your opinion. Cut meeting time in half with a working prototype.'],
+  ['Development', 'engineering', '/public/images/blogimg_all.png', 'Knowing what to build can be more important than how you build it. Don’t waste a second solving the wrong problem.'],
+  ['QA', 'production', '/public/images/blogimg_all.png', 'Development projects can often become bloated without an end in sight. We give you new functionality every three weeks, not once a year.'],
+  ['Support', 'support', '/public/images/blogimg_all.png', 'We understand that your success goes well beyond launch day. Our dedicated support model lets you focus on the broader operation.'],
+];
+
+class Services extends React.Component {
+  constructor(props: mixed, context: mixed): void {
+    super(props, context);
+
+    this.state = {target: null};
+    this.handleSelectSegment = this.handleSelectSegment.bind(this);
+  }
+
+  handleSelectSegment(event: mixed): void {
+    if (event) event.preventDefault();
+    var target = event.target.closest('a').getAttribute('href').replace('#', '');
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

this is a pretty sketchy API that will likely not work for most of our users. Is there a more declarative way we can handle it that doesn’t require DOM traversal?